### PR TITLE
CDAP-8708 Update exception message

### PR DESCRIPTION
**Pull request by rohitsinha54**

### CODE REVIEW

**Comment by rohitsinha54** [comment]
ran local build -DskipTests will not run a build since its just message change.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/ApplicationVerificationStage.java [hunk 1]
```diff
@@ -179,10 +179,11 @@ private void verifyOwner(NamespacedEntityId entityId, @Nullable KerberosPrincipa
       if (!Objects.equals(ownerPrincipal,
                          specifiedOwnerPrincipal == null ? null : specifiedOwnerPrincipal.getPrincipal())) {
         // Not giving existing owner information as it might be unacceptable under some security scenarios
-        throw new ConflictException(String.format("The specified %s : '%s' is not the same as existing one. " +
-                                                    "The %s of an %s cannot be updated.",
+        throw new ConflictException(String.format("%s '%s' already exists and the specified %s '%s' is not same as " +
```
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/ApplicationVerificationStage.java]: it is not _the_ same.

**Review by anew — COMMENTED** [review]
still a missing "the"

**Comment by anew** [comment]
LGTM

**Pull request merged**